c++/src/kj/exception.c++

```diff
@@ -1243,7 +1242,19 @@ static kj::String demangleTypeName(const char* name) {
 }
 
 kj::String getCaughtExceptionType() {
+  #if !defined (__EMSCRIPTEN__)
```

Comment:
Double-negatives are hard to think about. When an `if` has an `else`, I prefer to make the `if` branch always be the positive case to make it easier to read.